packages/blockchain-wallet-v4-frontend/src/scenes/Home/PriceIndexSeries/CoinTicker/selectors.js

```diff
@@ -4,10 +4,14 @@ import { lift } from 'ramda'
 
 export const getData = (state, coin) => {
   const settings = selectors.core.settings.getSettings(state)
-  const rates = coin === 'BTC'
-    ? selectors.core.data.bitcoin.getRates(state)
-    : coin === 'ETH' ? selectors.core.data.ethereum.getRates(state)
-      : selectors.core.data.bch.getRates(state)
+  const getRates = (coin) => {
+    switch (coin) {
+      case 'BTC': return selectors.core.data.bitcoin.getRates(state)
+      case 'ETH': return selectors.core.data.ethereum.getRates(state)
+      default: return selectors.core.data.bch.getRates(state)
```

Comment:
Not sure if the default should be BCH, for example, if theres a typo the rate will default to BCH. Maybe we should default to 0 and have a case for BCH?

Reply:
Definitely not a big fan of this logic. There is a similar switch after that one. It should return bch only when bch is requested. Otherwise it should probably fail